.devcontainer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM jeluard/substrate-playground-template-base@sha256:35683978140807d8e3c9eaa814e9034bd254e74224073984db2159302d44946b
```

Comment:
It will have to be updated every now and then. This is the base image that contains the rust toolchain and basic substrate tools. 
I will move to proper versioning later on.

Reply:
Does it contain more than https://hub.docker.com/r/paritytech/substrate-ci-linux ? Or can we just use that as our base image?

Reply:
For now it does as it prepares for some theia specific things but ultimately it could be any image.

Reply:
Id suggest implementing it as the base image and try to remove as much maintenance/versioning as possible for the time being-- could migrate to a theia specific image once necessary

Reply:
Unfortunately it is necessary now as this is how the playground interface (based on theia) shows up

Reply:
Okay, so let's leave it as it is for now, and @jeluard acknowledges that versioning the template base image should happen. I created a followup ticket for that. https://github.com/paritytech/substrate-playground/issues/145